### ISSUE TITLE
Training PSPNet

### ISSUE THREAD

**Issue by DonghyunK** [hszhao/PSPNet#23]
Hi,

I am trying to train PSPNet50. 

With a Titan X, I could only train it with the batch size of 1. I cannot train it with the batch size of 2 even using 2 Titan GPUs. 

Could please let me know how many gpus you used to train PSPNet and how many gpus are needed to train PSPNet successfully?
 

Thank you so much.

**Comment by wistone** [comment]
The experiments with ResNet101 are trained with 4 GPUs. Actually our training code contains many memory optimization so it needs less memory.

**Comment by DonghyunK** [comment]
@wistone Thank you for the reply.

Could you please let me know how do you optimize memory??

Thank you.

**Comment by wistone** [comment]
https://github.com/dmlc/mxnet-memonger

You can refer to the memory optimization in MXNet

**Comment by DonghyunK** [comment]
@wistone 

Did you train a model using MXNet and then convert the MXNet model into Caffe model??

Thank you.

**Comment by wistone** [comment]
We implement this method in our training platform in caffe.

**Comment by DonghyunK** [comment]
@wistone 

Is it publicly available? 

If not, could you please let me know how you implement this method in Caffe?

Thank you

**Comment by wistone** [comment]
No it is not public. You can read the paper for the theory, and refer the code in MXNet. It is doable.

**Comment by LearnerInGithub** [comment]
@hszhao @wistone  Could you explain what's mean of the three accuracy output on training phase? There only one SegAccuracy, so I think there should one accuracy output, why three ones occurs?

**Comment by balloch** [comment]
Wait, @LearnerInGithub , where did you get training files?!

**Comment by ThienAnh** [comment]
@DonghyunK  Can you public script for training?
